x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/transform/common.ts

```diff
@@ -4,8 +4,14 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { Logger } from 'src/core/server';
 import * as Registry from '../../registry';
+import { appContextService } from '../../../app_context';
 
 export const getAsset = (path: string): Buffer => {
   return Registry.getAsset(path);
 };
+
+export const getLogger = (): Logger => {
```

Comment:
Please remove this and instead import appContextService where it's needed to remain consistent with the rest of the plugin.